components/admin_console/banner.tsx

```diff
@@ -5,7 +5,12 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-export default function Banner(props) {
+type Props = {
+    title?: JSX.Element;
+    description: JSX.Element;
+}
+
+const Banner: React.FC<Props> = (props: Props) => {
```

Comment:
You're missing an `export default` here. Or at the bottom of the file.

Reply:
Thanks! Should be solved in the next commit.